inappmessaging/src/test/java/com/rakuten/tech/mobile/inappmessaging/runtime/view/InAppMessageViewListenerSpec.kt

```diff
@@ -193,4 +196,88 @@ class InAppMessageViewListenerSpec : BaseTest() {
 
         listener.onTouch(mockView, mockEvent).shouldBeFalse()
     }
+
+    @Test
```

Comment:
This test class is getting pretty long. Sometimes in this case I will split up the tests to have multiple classes within the same file. It helps the tests be more organized and easier to read. i.e. these new tests could be in a class named `InAppMessageViewListenerBackButtonSpec`. 

See here for an example of what I'm talking about: https://github.com/rakutentech/android-remote-config/blob/master/remote-config/src/test/kotlin/com/rakuten/tech/mobile/remoteconfig/RealRemoteConfigSpec.kt

Reply:
Created separate class for OnKey and OnTouch